sdk/README.md

```diff
@@ -138,53 +153,39 @@ will be listed below.
 - [Exit Handler](/sdk/FEATURES.md#exit-handler)
 
 
-## Additional Features
+## Removable Features
 
-### Compile Kubeflow Pipelines with Artifacts Enabled
+### Compile Kubeflow Pipelines without Artifacts
 
-**Prerequisite**: Install [Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/).
-
-By default, _artifacts_ are disabled because they are dependent on Kubeflow Pipeline's
+By default, _artifacts_ are enabled because the DSL are designed to run on Kubeflow Pipeline's
```

Comment:
```suggestion
By default, _artifacts_ are enabled because the KFP DSL are designed to run on Kubeflow Pipeline's engine with artifacts to be stored on
```